src/test/compile-fail/E0132.rs

```diff
@@ -11,7 +11,7 @@
 #![feature(start)]
 
 #[start]
-fn f<T>() {} //~ ERROR E0132
+fn f< T >() {} //~ ERROR E0132
```

Comment:
See: https://github.com/rust-lang/rust/pull/35264#discussion_r73522363

It definitely seems pretty minor imho